<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>578</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="b_w_theme">
      <property name="text">
       <string>Темная тема</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="custom_theme">
      <property name="text">
       <string>Настроить тему</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lang">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="help_documentation">
      <property name="text">
       <string>Помощь</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="about_author">
      <property name="text">
       <string>О разработчике</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>v0.0.0</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>331</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ok">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
